src/core.c/IO/Path.pm6

```diff
@@ -497,12 +497,13 @@ my class IO::Path is Cool does IO {
               :path($!os-path), :$uid, :$gid, :os-error(.Str) );
         }}
         my str $path = self.absolute;
+        my $stat := nqp::dispatch("boot-syscall", "file-stat", nqp::decont_s($path), 0);
```

Comment:
The `stat` here isn't technically needed unless `$uid.defined && $gid.defined && $uid ~~ UInt && $gid ~~ Uint`. Dunno if it's worth rewriting though.